flink-connectors/flink-connector-elasticsearch5/src/main/resources/META-INF/NOTICE

```diff
@@ -1,201 +1,104 @@
-OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
-WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
-
-===================================
-       Notice for scopt
-===================================
-
-This project bundles scopt (v. 3.5.0) underr the MIT License.
-
-Original source repository: https://github.com/scopt/scopt
-
-scopt - Copyright (c) scopt contributors
-
-See source files for details.
-
-Permission is hereby granted, free of charge, to any person obtaining
-a copy of this software and associated documentation files (the
-``Software''), to deal in the Software without restriction, including
-without limitation the rights to use, copy, modify, merge, publish,
-distribute, sublicense, and/or sell copies of the Software, and to
-permit persons to whom the Software is furnished to do so, subject to
-the following conditions:
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
 
-The above copyright notice and this permission notice shall be
-included in all copies or substantial portions of the Software.
+This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt)
+
+- com.carrotsearch:hppc:0.7.1
+- com.fasterxml.jackson.core:jackson-core:2.8.1
+- com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:2.8.1
+- com.fasterxml.jackson.dataformat:jackson-dataformat-smile:2.8.1
+- com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.8.1
+- com.github.spullara.mustache.java:compiler:0.9.3
+- com.tdunning:t-digest:3.0
+- commons-codec:commons-codec:1.10
+- commons-logging:commons-logging:1.1.3
+- io.netty:netty:3.10.6.Final
+- io.netty:netty-buffer:4.1.6.Final
+- io.netty:netty-codec:4.1.6.Final
+- io.netty:netty-codec-http:4.1.6.Final
+- io.netty:netty-common:4.1.6.Final
+- io.netty:netty-handler:4.1.6.Final
+- io.netty:netty-resolver:4.1.6.Final
+- io.netty:netty-transport:4.1.6.Final
+- joda.time:joda-time:2.5
+- net.java.dev.jna:jna:4.2.2
+- org.apache.logging.log4j:log4j-api:2.7
+- org.apache.logging.log4j:log4j-to-slf4j:2.7
+- org.apache.lucene:lucene-core:6.3.0
+- org.apache.lucene:lucene-analyzers-common:6.3.0
+- org.apache.lucene:lucene-backward-codecs:6.3.0
+- org.apache.lucene:lucene-grouping:6.3.0
+- org.apache.lucene:lucene-highlighter:6.3.0
+- org.apache.lucene:lucene-join:6.3.0
+- org.apache.lucene:lucene-memory:6.3.0
+- org.apache.lucene:lucene-misc:6.3.0
+- org.apache.lucene:lucene-queries:6.3.0
+- org.apache.lucene:lucene-queryparser:6.3.0
+- org.apache.lucene:lucene-sandbox:6.3.0
+- org.apache.lucene:lucene-spatial:6.3.0
+- org.apache.lucene:lucene-spatial-extras:6.3.0
+- org.apache.lucene:lucene-spatial3d:6.3.0
+- org.apache.lucene:lucene-suggest:6.3.0
+- org.apache.httpcomponents:httpasynclcient:4.1.2
+- org.apache.httpcomponents:httpclient:4.5.3
+- org.apache.httpcomponents:httpcore:4.4.6
+- org.apache.httpcomponents:httpcore-nio:4.4.5
```

Comment:
not properly sorted. `org.apache.httpcomponents` should be in front of `org.apache.logging.`